media/wav.ksy

```diff
@@ -26,11 +45,19 @@ doc: |
   followed by a sequence of data chunks. A WAVE file is often just a RIFF
   file with a single "WAVE" chunk which consists of two sub-chunks --
   a "fmt " chunk specifying the data format and a "data" chunk containing
-  the actual sample data.
+  the actual sample data, although other chunks exist and are used.
+
+  An extension of the file format is the Broadcast Wave Format for radio
+  broadcasts. Sample files can be found at:
+
+  https://www.bbc.co.uk/rd/publications/saqas
 
   This Kaitai implementation was written by John Byrd of Gigantic Software
   (jbyrd@giganticsoftware.com), and it is likely to contain bugs.
-doc-ref: http://soundfile.sapp.org/doc/WaveFormat/
+doc-ref:
+  - http://soundfile.sapp.org/doc/WaveFormat/
+  - https://web.archive.org/web/20101031101749/http://www.ebu.ch/fr/technical/publications/userguides/bwf_user_guide.php
+  - https://www.itu.int/rec/R-REC-BS.2076-2-201910-I/en
```

Comment:
@armijnhemel This only describes the `chna` chunk which is not even described yet, right? I would move it to the particular type (or enum key if there is no type yet) that it describes, because currently it's confusing - if it's in the top-level `/doc-ref` without any further description, I'd expect that it documents the overall WAV/BWF format and/or many of its chunks, not just a single one.

Reply:
It is described in chapter 7 of https://www.itu.int/dms_pubrec/itu-r/rec/bs/R-REC-BS.2076-2-201910-I!!PDF-E.pdf

Moving it to the enum makes perfect sense.